Sources/RealDeviceMapLib/Modell/SpawnPoint.swift

```diff
@@ -109,6 +117,37 @@ public class SpawnPoint: JSONConvertibleObject {
 
     }
 
+    public static func setLastSeen(mysql: MySQL?=nil, spawnId: UInt64, oldLastSeen: UInt32) throws {
+
+        let now = UInt32(Date().timeIntervalSince1970)
+
+        if oldLastSeen + 3600 > now {
+            return
```

Comment:
TODO: Test if we hit this check.

Reply:
why do you distinguish between lastSeen and updated? should be almost the same timestamp, isn't it?
you can update the `save` method to improve the save logic